Scripts/assignment/emme_bindings/mock_project.py

```diff
@@ -0,0 +1,423 @@
+import numpy
+from collections import namedtuple
+
+
+class MockProject:
+    """Mock-up version of `EmmeProject`.
+
+    Emulates key features of EMME API, so that `EmmeAssignmentModel`
+    and `AssignmentPeriod` can be unit tested without EMME binding.
+    """
+
+    path = "c:\\xxx\\yyy"
```

Comment:
What is the meaning of this `path`?

Reply:
In `EmmeProject`, it refers to the directory path of the *emmebank* file and it is used to locate the function files here: https://github.com/HSLdevcom/helmet-model-system/blob/8d6073468da8c7519fd575edbd809e5804219707/Scripts/assignment/assignment_period.py#L460

I will remove this dependency in the Helmet 4.1 development.